api/src/main/resources/Reference_Application_Diagnoses-11.xml

```diff
@@ -1,240 +1,240 @@
+  <concept concept_id="1293" retired="false" datatype_id="4" class_id="4" is_set="false" creator="1" date_created="2006-04-22 12:13:15.0" version="" changed_by="1" date_changed="2016-05-27 03:19:44.0" retired_by="1" uuid="1293AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"/>
+  <concept concept_id="1490" retired="false" datatype_id="4" class_id="4" is_set="false" creator="1" date_created="2007-07-13 15:47:22.0" version="" changed_by="1" date_changed="2015-07-24 01:14:19.0" retired_by="1" uuid="1490AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"/>
+  <concept concept_id="5018" retired="false" datatype_id="4" class_id="12" is_set="false" creator="1" date_created="2004-08-12 00:00:00.0" version="564524" changed_by="1" date_changed="2016-02-26 02:24:33.0" retired_by="1" uuid="5018AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"/>
+  <concept concept_id="5258" retired="false" datatype_id="4" class_id="4" is_set="false" creator="1" date_created="2004-08-12 00:00:00.0" version="" changed_by="1" date_changed="2016-05-27 02:50:42.0" retired_by="1" uuid="5258AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"/>
+  <concept concept_id="5340" retired="false" datatype_id="4" class_id="4" is_set="false" creator="1" date_created="2004-08-12 00:00:00.0" version="" changed_by="1" date_changed="2013-06-01 19:46:15.0" retired_by="1" uuid="5340AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"/>
+  <concept concept_id="5622" retired="false" datatype_id="4" class_id="11" is_set="false" creator="1" date_created="2004-08-12 00:00:00.0" version="" changed_by="1" date_changed="2016-02-20 05:04:58.0" retired_by="1" uuid="5622AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"/>
+  <concept concept_id="110598" retired="false" datatype_id="4" class_id="4" is_set="false" creator="1" date_created="2007-11-03 00:00:00.0" version="" changed_by="1" date_changed="2011-06-09 00:40:16.0" retired_by="1" uuid="110598AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"/>
+  <concept concept_id="110834" retired="false" datatype_id="4" class_id="4" is_set="false" creator="1" date_created="2007-11-03 00:00:00.0" version="" changed_by="1" date_changed="2015-11-03 02:56:58.0" retired_by="1" uuid="110834AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"/>
+  <concept concept_id="111633" retired="false" datatype_id="4" class_id="4" is_set="false" creator="1" date_created="2007-11-03 00:00:00.0" version="" changed_by="1" date_changed="2015-11-03 02:53:04.0" retired_by="1" uuid="111633AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"/>
+  <concept concept_id="111636" retired="false" datatype_id="2" class_id="4" is_set="false" creator="1" date_created="2007-11-03 00:00:00.0" version="" changed_by="1" date_changed="2016-11-27 22:21:19.0" retired_by="1" uuid="111636AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"/>
+  <concept concept_id="112198" retired="false" datatype_id="4" class_id="4" is_set="false" creator="1" date_created="2007-11-03 00:00:00.0" v
```

Comment:
@dkayiwa  A whole bunch of these changes is due to this new column called retired_by. Should we have it?

Reply:
Why would be need a retired_by when retired="false"?

Reply:
Exactly

Reply:
Exactly

Reply:
I guess we'll remove it

Reply:
So can you?

Reply:
@dkayiwa Yes, was working on it. Just committed the changes. Do check out :)